vehicle/learning_based_vehicle_calibration/launch/calibration_launch.py

```diff
@@ -0,0 +1,184 @@
+import os
+
+import launch
+from launch.actions import OpaqueFunction
+import launch.substitutions
+import launch_ros.actions
+
+
+def launch_data_monitor(context):
+    # Open a new terminal and run data_monitor.py
+    os.system(
+        "gnome-terminal -- /bin/bash -c 'ros2 run learning_based_vehicle_calibration data_monitor.py; exec bash'"
```

Comment:
```suggestion
        "gnome-terminal -- /bin/bash -c 'source $HOME/autoware/install/setup.bash && ros2 run learning_based_vehicle_calibration data_monitor.py; exec bash'"
```

Since this opens up a new terminal, autoware workspace should be sourced here again.

And this should be a step in the readme file.